### PR TITLE
HAAR-888 Add users/me (current user details) endpoint

### PR DESCRIPTION
**Pull request by karenmillermoj**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/UserController.kt [hunk 1]
```diff
@@ -58,6 +59,37 @@ class UserController(
...
+    summary = "My User details.",
+    description = "Find my user details."
+  )
+  @ApiResponses(
+    value = [
+      ApiResponse(
+        responseCode = "200",
+        description = "OK",
+        content = [
+          io.swagger.v3.oas.annotations.media.Content(
```
Comment (ravmoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/UserController.kt:73]: import is already there : io.swagger.v3.oas.annotations.media

**Review by ravmoj — APPROVED** [review]
lgtm

File: src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/UserController.kt (hunk 1 above)
Comment (karenmillermoj) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/UserController.kt:73]: doh!

**Review by ravmoj — APPROVED** [review]
lgtm

**Pull request merged**